language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/codeaction/CodeActionTest.java

```diff
@@ -164,6 +203,14 @@ public void testCodeActionWithDiagnostics(String config, String source) throws I
         };
     }
 
+    @DataProvider(name = "codeaction-testgen-data-provider")
+    public Object[][] testGenCodeActionDataProvider() {
+        return new Object[][]{
+                {"testGenFunctionCodeAction.json", Paths.get("testgen", "module1", "functions.bal")},
+//                {"testGenServiceCodeAction.json", Paths.get("testgen", "module2", "services.bal")}
```

Comment:
remove commented out code section

Reply:
Thanks! fixed in https://github.com/ballerina-platform/ballerina-lang/pull/11447